### PR TITLE
UI text upgrades

### PR DESCRIPTION
**Pull request by it33**
- Allow anyone to join this team > Allow any authenticated user to join this team
- Re-generate > Regenerate

### CODE REVIEW

**Comment by enahum** [comment]
+1

File: webapp/components/team_general_tab.jsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const holders = defineMessages({
     },
     openInviteTitle: {
         id: 'general_tab.openInviteTitle',
-        defaultMessage: 'Allow anyone to join this team'
+        defaultMessage: 'Allow any authenticated user to join this team'
```
Comment (crspeller) [inline webapp/components/team_general_tab.jsx]: Using `authenticated` here sounds like you need to authenticate to the team. Maybe just drop the word? So `Allow any user to join this team`.

Comment (it33) [inline webapp/components/team_general_tab.jsx]: @crspeller, this is coming from a customer who thought "Allow any user to join this team" meant unauthenticated users. How about "Allow any user with an account on this server to join this team"?

Comment (crspeller) [inline webapp/components/team_general_tab.jsx]: @it33 More verbose but sounds good.

**Comment by it33** [comment]
@crspeller PR updated to "Allow any user with an account on this server to join this team"

**Comment by crspeller** [comment]
+1

**Pull request merged**